Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -176,15 +201,13 @@ class SnackBar: UIView {
             }
         })
 
+        // Now we adjust the snackbars height to make sure it wraps everything tightly.
         self.snp_makeConstraints({ make in
```

Comment:
Same as the comment above. If we have text that could potentially grow, we shouldn't set a height constraint on the snackbar view from inside the snackbar. The parent view should be responsible for setting up the widget's height.